### PR TITLE
Show some invisible detail for easy analysis.

### PR DESCRIPTION
**Pull request by Sunev**
By setting ```log.to``` as -1, these logs will be invisible until game end.

### CODE REVIEW

**Comment by Sunev** [comment]
I've got a bug report as that a 「Cat×Cat」 posioned only one player in https://github.com/Sunev/jinrou/issues/41 .
It took me 1 hour to find out that the other posion was defeated by a Miko.

**Comment by uhyo** [comment]
I really like this feature, but I'd like to give them a new log type apart from `"skill"` or `"system"`, not to confuse players.
Maybe `"hidden"` would be good?

Adding a new log type is rather hard as the front-end (TypeScript) part is involved, so I'm willing to take it. Or, if you're interested, I can instruct you.

Besides, I'd like some fix of Japanese text...

File: language/ja/roles.yaml [hunk 1]
```diff
@@ -199,6 +199,10 @@ Guard:
   select: "{{name}}は{{target}}を護衛しました。"
   # Log of GJ
   gj: "{{guard}}が{{name}}の護衛に成功しました。"
+  
+# Poisoner
+Poisoner:
+  select: "{{name}}の毒薬が{{target}}にかけだ。"
```
Comment (uhyo) [inline language/ja/roles.yaml]: Poisoner's poison is not necessarily "毒薬", and "かけだ" is not valid Japanese. I propose the following:

`{{name}}の毒が{{target}}を襲いました。`

File: language/ja/roles.yaml [hunk 2]
```diff
@@ -302,6 +306,7 @@ Spellcaster:
 Priest:
   # Selected a target.
   select: "{{name}}が{{target}}を聖なる力で守りました。"
+  protected: "{{name}}は聖なる力で{{found}}に抵抗した。"
```
Comment (uhyo) [inline language/ja/roles.yaml]: Logs should use 敬語, so `"{{name}}は聖なる力で{{found}}に抵抗しました。"` is better.

File: language/ja/roles.yaml [hunk 3]
```diff
@@ -434,6 +439,7 @@ Counselor:
 Miko:
   # Select a self protection.
   select: "{{name}}が聖なる力で自身を守りました。"
+  protected: "{{name}}は聖なる力で{{found}}に抵抗した。"
```
Comment (uhyo) [inline language/ja/roles.yaml]: Same as above.

`"{{name}}は聖なる力で{{found}}に抵抗しました。"`

File: language/ja/game.yaml [hunk 4]
```diff
@@ -216,6 +216,36 @@ found:
...
+  friendsuicide: "恋人の後を追って自殺"
+  twinsuicide: "双子の後を追って自殺"
+  infirm: "老衰"
+  hunter: "銃殺"
+  gmpunish: "ゲームマスター"
+  gone-day: "昼間突然死"
+  gone-night: "夜間突然死"
+  crafty: "死んだふり"
+  fallback: "不明"
+  situation: "{{name}}の死因は{{detail}}です。"
```
Comment (uhyo) [inline language/ja/game.yaml]: The text makes sense, but I prefer past form:

`"{{name}}の死因は{{detail}}でした。"`

**Comment by Sunev** [comment]
Thank you for your Japanese knowledge.
And I don't really want to touch the front-end code. Please feel free to take it.

**Comment by uhyo** [comment]
Great! Then I'll take it. Thank you for this PR.

**Comment by hakoniwawind** [comment]
foundDetail変更案（個人的な趣味）
poison: "毒薬"→"毒"
witch: "魔女的毒薬"→"毒薬"
hinamizawa: "雛見沢症候群"→"感染症"
　内部ID的に元ネタまるわかりですが…直球で良いのでしょうか？
spygone: "行方不明"→or"失踪"？
　死因は…と続くと難しいですね；
　サンタクロースも同じIDを使用しているので、そこも考慮が必要か。
foxsuicide: "妖狐の後を追って自殺"→"妖狐の後追い自殺"
friendsuicide: "恋人の後を追って自殺"→"恋人の後追い自殺"
twinsuicide: "双子の後を追って自殺"→"双子の後追い自殺"
gmpunish: "ゲームマスター"→"ゲームマスターの宣告"…とか？

**Pull request merged**